lib/oxidized/worker.rb

```diff
@@ -44,13 +44,15 @@ def process job
         msg = "update #{node.name}"
         msg += " from #{node.from}" if node.from
         msg += " with message '#{node.msg}'" if node.msg
-        output = node.output.new
-        if output.store node.name, job.config,
-                              :msg => msg, :user => node.user, :group => node.group
-          Oxidized.logger.info "Configuration updated for #{node.group}/#{node.name}"
-          Oxidized.Hooks.handle :post_store, :node => node,
-                                             :job => job,
-                                             :commitref => output.commitref
+        node.output.each do |output|
+          output = node.output.new
```

Comment:
not really sure this code do what you want. 😕 

you receive a `output` variable from the block `#each` but you shadow it here. if I guess I think you mean something like:

``` ruby
node.output.each do |output|
  current_output = output.new
  // ...
end
```